CHANGELOG.md

```diff
@@ -211,6 +211,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - CPU usage spikes due to mouse movements for unfocused windows on X11/Windows
 - First window on macOS not tabbed with system prefer tabs setting
 - Window being treaten as focused by default on Wayland
+- `window.startup_mode` applied to window again when creating new tab
```

Comment:
I'm **really** confused how you managed to put this changelog entry so far away from the latest release.

Reply:
Me too...